composer.json

```diff
@@ -24,7 +24,7 @@
         }
     },
     "require": {
-        "statamic/cms": "3.0.*"
+        "statamic/cms": "^3.0 || ^3.1"
```

Comment:
It should be `"3.0.* || 3.1.*"`

Reply:
Done! Out of curiosity, what's the difference between them?

Reply:
https://getcomposer.org/doc/articles/versions.md#caret-version-range-

`^3.0` would allow anything below `4.0`. Since Statamic doesn't use semantic versioning, we don't want you to be able to use the ssg package on 3.2, 3.3, etc until it's actually ready. Yeah you'd be able to install it, but it's possible it'll be broken. (This also means `^3.0 || ^3.1` is redundant, both are basically saying the same thing)

`3.0.*` would allow any 3.0.x version.
`3.0.* || 3.1.*` would allow any 3.0 or 3.1 version. 3.2 won't be allowed.

Reply:
Ah gotcha, okay! Makes sense (I should probably update all my addons to be that way haha).

Reply:
@duncanmcclean request review again  :laughing: